lyx-sh.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+directory="${HOME}/.lyx"
+_pwd=$PWD
+
+Init(){
+	
+	echo "sh --> Starting LyX ..."
+	lyx
+}
+
+Prepare() {
+	
+		echo "sh --> Directory $directory is empty, executing: configure.py"
+		cd $directory
+		python3 /app/share/lyx/configure.py
+		echo "sh --> Copying lyxrc.defaults ..."
+		cp /app/temp/lyxrc.defaults ${directory}/
+		cd $_pwd
+		
+		Init
+    
+}
+
+if [ -d "$directory" ]; then
+
+    if test -z "$(ls -A $directory)"; then
+		Prepare
+    else
+        Init
+    fi
+
+else
+    mkdir $directory
+    Prepare
```

Comment:
here too. you should really start lyx.

Reply:
What happens is that the program does not work properly if it is started immediately, it generates a faulty configuration, that is why the file is copied to ./lyx

Reply:
Drop the else and just run lyx if the directory exists.

Reply:
> What happens is that the program does not work properly

But what happens right now is that it won't start on first try. This is poor user experience.

Note that this whole behaviour is a bug upstream. Apps that require that kind of operation are just broken.